### PR TITLE
update version of command-line-args which also now requires command-line-usage

### PR DESCRIPTION
**Pull request by zzo**
can now work with node6

### CODE REVIEW

**Comment by zzo** [comment]
I signed it!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymer-css-build",
-  "version": "0.0.6",
+  "version": "0.0.7",
```
Comment (rictic) [inline package.json]: Don't update the version number in a pull request, we'll do that when we stamp a new release based on any other unreleased changes.

Comment (zzo) [inline package.json]: ok done

**Comment by azakus** [comment]
Hmm, not sure why the tests are failing, but LGTM

**Pull request merged**